egs/multi_zh-hans/ASR/README.md

```diff
@@ -0,0 +1,23 @@
+
+# Introduction
+
+This recipe includes scripts for training Zipformer model using multiple Chinese datasets.
+
+# Included Training Sets
```

Comment:
Could you make a table like below:
|Datset| Number of hours| URL|
|---|---:|---|
|THCHS-30|35|https://www.openslr.org/18/|

Reply:
OK!